tests/functional/cephx_keyrotation/test_cephx_keyrotation_mismatch_alert.py

```
@@ -0,0 +1,1042 @@
+        )
+
+
+@skipif_external_mode
+@skipif_ocs_version("<4.22")
+@green_squad
+class TestCephXCephClientAnnotations:
+    """TC14–TC15: CephClient created-with-cephx-features and keyType."""
+
+    @tier1
+    def test_cephclients_have_cephx_annotation_and_aes_keytype(
+        self, cephx_bootstrap_setup
```

Ruff (0.15.21)

[warning] Unused method argument: `cephx_bootstrap_setup`

(ARG002)